Source/Core/DolphinLibretro/Input.cpp

```diff
@@ -602,24 +630,58 @@ void retro_set_controller_port_device(unsigned port, unsigned device)
         ncShake->SetControlExpression(1, "L2");                       // Y
         ncShake->SetControlExpression(2, "L2");                       // Z
 
-        wmButtons->SetControlExpression(2, "Start");   // 1
-        wmButtons->SetControlExpression(3, "Select");  // 2
-        wmButtons->SetControlExpression(4, "L");       // -
-        wmButtons->SetControlExpression(5, "R");       // +
+        wmButtons->SetControlExpression(2, "Start");                         // 1
+        wmButtons->SetControlExpression(3, "Select");                        // 2
+        wmButtons->SetControlExpression(4, "L");                             // -
+        wmButtons->SetControlExpression(5, "R");                             // +
```

Comment:
maybe move the 0/1 expressions in here too? since you are overriding them anyway in the `else` block.